latch_cli/utils/__init__.py

```diff
@@ -99,7 +114,24 @@ def current_workspace() -> str:
     ws = user_config.workspace_id
     if ws == "":
         ws = account_id_from_token(retrieve_or_login())
-        user_config.update_workspace(ws, "Personal Workspace")
+        ws = execute(
+            gql.gql("""
+            query GetUserDefaultWorkspace($accountId: BigInt!) {
+                userInfoByAccountId(accountId: $accountId) {
+                    defaultAccount
+                }
+            }"""),
+            {"accountId": ws},
+        )["userInfoByAccountId"]["defaultAccount"]
+
+        workspace_names = get_workspaces()
+
+        if ws not in workspace_names:
+            raise ValueError(
+                f"Default workspace {ws} not found in the list of workspaces."
+            )
+
+        user_config.update_workspace(ws, workspace_names[ws])
```

Comment:
default workspace should auto-update unless you explicitly overwrote it

so i'd just remove it